spikes/harbor/README.md

```diff
@@ -0,0 +1,255 @@
+
+# Deploy Harbor
+
+## Azure ASB Cluster
+
+> The steps below are guidelines for the dev cluster, but is replicable for the pre-prod as well
+
+- Create a dns entry for harbor (*harbor-core-westus2-dev.cse.ms*) `rg-ngsa-asb-dev/cse.ms` private dns and put the Load-Balancer IP address for Westus2.
+- Create a public dns entry `harbor-core-westus2-dev.cse.ms` in `dns-rg/cse.ms` and put the public ip for WestUS2 dev cluster
+- Add below settings for harbor (similar to ngsa apps) in Application Gateway:
+  - Backend pool
+  - Backend settings (with no custom probes, that we don't have to configure probes)
+  - Listeners (http and https)
+  - Rules (http and https)
+- Add an exception for the harbor-host in the WAF policy (e.g. for WestUS2-dev you'll add it in `rg-ngsa-asb-dev/ngsa-asb-waf-policy-westus2`)
+
+At this point Harbor should be ready to deploy.
+Now we need to make sure our cluste can pull the container images.
+To do that we have two options:
+
+- We can push Harbor images into our cluster acr repo (`rg-ngsa-asb-dev/acraks3i2qzkkxofr7c`)
+    > This is easier and preferable for pre-prod and dev clusters
+- Or we can use another private repo and make sure our cluster can access the repo
+    > This requires several extra steps and should be done for SPIKEs only
+
+Extra steps for deploying using another private repo (in this case `kdevacr.azurecr.io`) couple of things to keep in mind:
+
+- Add the ACR's address in `rg-ngsa-asb-dev-hub/fw-policies-eastus` allow rule collection.
+- Add managed identity for the Westus2 cluster (e.g `aks-3i2qzkkxofr7c-westus2-agentpool`) to the ACR and give it `AcrPull` permissions
+- Deploy helm in `azure-arc` repo, since its in the policy exception list.
+
+Common steps:
+
+- Change the repository values (13 of time) in the helm chart to point to the proper acr repo
```

Comment:
Documented steps to change repository value.
Original Comment: https://github.com/retaildevcrews/istio/pull/68#discussion_r932346753